internal/controllers/vgmanager/lvm/lvm.go

```diff
@@ -249,9 +249,9 @@ func (hlvm *HostLVM) DeleteVG(vg VolumeGroup) error {
 	}
 
 	for _, pv := range vg.PVs {
-		_, err = hlvm.ExecuteCommandWithOutput(lvmDevicesCmd, "--delpvid", pv.UUID)
+		_, err = hlvm.ExecuteCommandWithOutputAsHost(lvmDevicesCmd, "--delpvid", pv.UUID)
```

Comment:
Why do we need to run this as host now?

Reply:
it was previously not run as host by accident, this command should never be run with less privileges than the others. It accesses the lvmdevices file